docs/shared/react-standalone-tutorial/2-project-graph.md

```diff
@@ -0,0 +1,300 @@
+# React Tutorial - Part 2: Project Graph
+
+Run the command: `npx nx graph`. A browser should open up with the following contents:
+
+{% graph height="450px" %}
+
+```json
+{
+  "hash": "85fd0561bd88f0bcd8703a9e9369592e2805f390d04982fb2401e700dc9ebc59",
+  "projects": [
+    {
+      "name": "cart",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "shared-ui",
+      "type": "lib",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "e2e",
+      "type": "e2e",
+      "data": {
+        "tags": []
+      }
+    },
+    {
+      "name": "store",
+      "type": "app",
+      "data": {
+        "tags": []
+      }
+    }
+  ],
+  "dependencies": {
+    "cart": [],
+    "shared-ui": [],
+    "e2e": [{ "source": "e2e", "target": "store", "type": "implicit" }],
+    "store": []
+  },
+  "workspaceLayout": { "appsDir": "apps", "libsDir": "libs" },
+  "affectedProjectIds": [],
+  "focus": null,
+  "groupByFolder": false,
+  "exclude": []
+}
+```
+
+{% /graph %}
+
+This is still different than the design from the start of Part 1:
```

Comment:
I think it's a bit weird that the original graph is the design. I can sort of interpret the design was to have shared components in a separate library or something like that. 

I know that Zack's tutorials frame it like this, but it looks off. It's backwards from what the actual thinking process it. No one is thinking: I need to have Graph A so I'm going to make the changes such that Graph A appears :) It's more like: i need to share some components. Let me do it. Oh look, it looks like this now.

If you said something like:

The graph is derived from the source code. Because the libraries don't use shared components yet, there are no dependencies between them. Let's change our code to use the components and then check how the graph changes.